src/convnets/vgg.jl

```diff
@@ -128,9 +129,9 @@ end
 @functor VGG
 
 function VGG(depth::Integer; pretrain::Bool = false, batchnorm::Bool = false,
-             inchannels::Integer = 3, nclasses::Integer = 1000)
+             inchannels::Integer = 3, nclasses::Integer = 1000, dropout_prob = 0.5)
     _checkconfig(depth, keys(VGG_CONFIGS))
-    layers = vgg((224, 224); config = VGG_CONFIGS[depth], batchnorm, inchannels, nclasses)
+    layers = vgg((224, 224); config = VGG_CONFIGS[depth], batchnorm, inchannels, nclasses, dropout_prob)
```

Comment:
Not sure if we should introduce `dropout_prob` for the `VGG` function. Aside from the fact that this would be breaking, Metalhead has the `vgg` function for exactly this sort of customisation. What do you think @darsnack?

Reply:
Xref https://github.com/FluxML/Metalhead.jl/issues/271#issuecomment-1892604680 for a way to load weights without this, but maybe this is worth it if only to avoid that extra code?

Reply:
It's not breaking as long as the default probability matches pre-#270. But I agree that it's better to leave it off for API consistency.

Reply:
Addressed in [69b34c3](https://github.com/FluxML/Metalhead.jl/pull/272/commits/69b34c3dc386ae63b3e4efa1f791518e2bfe8618) and [be19ebf](https://github.com/FluxML/Metalhead.jl/pull/272/commits/be19ebf6054c9f73ff74e7a30a7f4b66d28e2e86).